src/content/documentation/develop/ux-guidelines-for-mobile-extensions.md

```diff
@@ -105,23 +108,28 @@ Pass your colors through WebAIM’s [Color Contrast Checker](https://webaim.org/
 
 Match the font families used in Firefox for Android:
 
-- [Inter](https://fonts.google.com/specimen/Inter) for body and capition context
+- [Inter](https://fonts.google.com/specimen/Inter) for body and caption context
 - [Metropolis](https://www.1001fonts.com/metropolis-font.html) for headlines, subtitles, and buttons
 
+<!--
 #### Type scale
 
-Follow the [Firefox for Android typographic scale](https://www.figma.com/proto/TebIgESfWmQkMcEmwNaYZl/Android?node-id=0%3A806&viewport=115%2C161%2C0.5&scaling=min-zoom).
+Follow the [Firefox for Android typographic scale](https://www.figma.com/proto/TebIgESfWmQkMcEmwNaYZl/Android?node-id=0%3A806&viewport=115%2C161%2C0.5&scaling=min-zoom). 
+
+This link requires figma login, any idea where it should go?
+
+-->
```

Comment:
**note:** For reference I captured a screenshot of this figma too (e.g. to reference it in the issue we will have to file anyway) and pasted in the collapsible details below.

<details>

![image](https://github.com/mozilla/extension-workshop/assets/11484/8ee70b80-ac5f-4362-b53c-bf62314a91c5)


</details>

Reply:
I'd drop this link too.

Reply:
See https://github.com/mozilla/extension-workshop/issues/1659